dashboard/config/routes.rb

```diff
@@ -350,6 +350,7 @@
         post 'update_exemplar_code'
         get 'level_properties'
         get 'extra_links'
+        patch 'update_bubble_choice_settings'
```

Comment:
Style points for using patch here 😎, I wouldn't have objected to a POST, but PATCH is more subtly right in a way that is nice.